### DISCUSSION TITLE
How do RHF issues work?

### DISCUSSION THREAD

**Discussion by markwoon** [react-hook-form/react-hook-form#8648] [General] [answered]
What exactly is the workflow for GH issues for RHF?

For example, I created a feature request, and it was closed and "pinned to project board".

To me, "closed" mean nothing else is going to happen, but this implies that it's not dead.

So what's the lifecycle of an issue for RHF?

**Comment by Moshyfawn** [comment, marked as answer]
The feature requests get pinned to the project board for triage: waiting for other community members to up-vote. If it gets a considerable amount of up-votes, the RHF teams takes into work and.. well work on it.

Closing the issues doesn't mean won't be worked on, so don't worry

**Reply by Moshyfawn** [reply]
Here's the official post about RHF features requests: [the post](https://github.com/react-hook-form/react-hook-form/issues/1981#issue-646826298)

**Reply by markwoon** [reply]
That doesn't really explain why the issue is closed.  It can go on the project board and remain open.  At least then the issue's status will be an accurate reflection of the state of the issue.

**Reply by bluebill1049** [reply]
Thanks for the feedback. Honestly, I really want to give everyone a perfect solution for maintaining a library, but remember each project or library could have its own way of maintenance. How we maintain it may not be in your favor, but that doesn't mean it's not been maintained or feature requests are not been actioned (probably the opposite). I wish there are a separate board for feature requests, bugs and etc. At least, that's the way I would prefer to manage an open-source project or actually any project, so things can be categorized. The way it's been maintained (for the past 3 years) will stay as it is and probably will be for the incoming years.

**Reply by IanVS** [reply]
>Closing the issues doesn't mean won't be worked on, so don't worry

It looks like issues are now being locked, so it's not possible to upvote on them anymore.  What is the suggested way to voice support for closed and locked issues?  Should new issues / feature requests be created?
